test_unstructured/test_utils.py

```diff
@@ -105,7 +105,7 @@ def test_only_gives_only(iterator):
 @pytest.mark.parametrize("iterator", [[0, 1], (0, 1), range(10)])
```

Comment:
There are a lot of type errors in this file. Want to use this as an opportunity to clean those up?

Reply:
Sounds good. I was unsure about including that in the scope of this PR, but will do :)